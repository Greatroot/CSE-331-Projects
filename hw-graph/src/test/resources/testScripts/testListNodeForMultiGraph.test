# Tests listing all nodes within a 100 node graph

CreateGraph graph
AddNode graph n1
AddNode graph n2
AddNode graph n3
AddNode graph n4
AddNode graph n5
AddNode graph n6
AddNode graph n7
AddNode graph n9
AddNode graph n10
AddNode graph n11
AddNode graph n12
AddNode graph n13
AddNode graph n14
AddNode graph n15
AddNode graph n16
AddNode graph n17
AddNode graph n18
AddNode graph n19
AddNode graph n20
AddNode graph n21
AddNode graph n22
AddNode graph n23
AddNode graph n24
AddNode graph n25
AddNode graph n26
AddNode graph n27
AddNode graph n28
AddNode graph n29
AddNode graph n30
AddNode graph n31
AddNode graph n32
AddNode graph n33
AddNode graph n34
AddNode graph n35
AddNode graph n36
AddNode graph n37
AddNode graph n38
AddNode graph n39
AddNode graph n40
AddNode graph n41
AddNode graph n42
AddNode graph n43
AddNode graph n44
AddNode graph n45
AddNode graph n46
AddNode graph n47
AddNode graph n48
AddNode graph n49
AddNode graph n50
AddNode graph n51
AddNode graph n52
AddNode graph n53
AddNode graph n54
AddNode graph n55
AddNode graph n56
AddNode graph n57
AddNode graph n58
AddNode graph n59
AddNode graph n60
AddNode graph n61
AddNode graph n62
AddNode graph n63
AddNode graph n64
AddNode graph n65
AddNode graph n66
AddNode graph n67
AddNode graph n68
AddNode graph n69
AddNode graph n70
AddNode graph n71
AddNode graph n72
AddNode graph n73
AddNode graph n74
AddNode graph n75
AddNode graph n76
AddNode graph n77
AddNode graph n78
AddNode graph n79
AddNode graph n80
AddNode graph n81
AddNode graph n82
AddNode graph n83
AddNode graph n84
AddNode graph n85
AddNode graph n86
AddNode graph n87
AddNode graph n88
AddNode graph n89
AddNode graph n90
AddNode graph n91
AddNode graph n92
AddNode graph n93
AddNode graph n94
AddNode graph n95
AddNode graph n96
AddNode graph n97
AddNode graph n98
AddNode graph n99
AddNode graph n100
ListNodes graph