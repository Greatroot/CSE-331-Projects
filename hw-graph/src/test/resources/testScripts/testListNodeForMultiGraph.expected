# Tests listing all nodes within a 100 node graph

created graph graph
added node n1 to graph
added node n2 to graph
added node n3 to graph
added node n4 to graph
added node n5 to graph
added node n6 to graph
added node n7 to graph
added node n9 to graph
added node n10 to graph
added node n11 to graph
added node n12 to graph
added node n13 to graph
added node n14 to graph
added node n15 to graph
added node n16 to graph
added node n17 to graph
added node n18 to graph
added node n19 to graph
added node n20 to graph
added node n21 to graph
added node n22 to graph
added node n23 to graph
added node n24 to graph
added node n25 to graph
added node n26 to graph
added node n27 to graph
added node n28 to graph
added node n29 to graph
added node n30 to graph
added node n31 to graph
added node n32 to graph
added node n33 to graph
added node n34 to graph
added node n35 to graph
added node n36 to graph
added node n37 to graph
added node n38 to graph
added node n39 to graph
added node n40 to graph
added node n41 to graph
added node n42 to graph
added node n43 to graph
added node n44 to graph
added node n45 to graph
added node n46 to graph
added node n47 to graph
added node n48 to graph
added node n49 to graph
added node n50 to graph
added node n51 to graph
added node n52 to graph
added node n53 to graph
added node n54 to graph
added node n55 to graph
added node n56 to graph
added node n57 to graph
added node n58 to graph
added node n59 to graph
added node n60 to graph
added node n61 to graph
added node n62 to graph
added node n63 to graph
added node n64 to graph
added node n65 to graph
added node n66 to graph
added node n67 to graph
added node n68 to graph
added node n69 to graph
added node n70 to graph
added node n71 to graph
added node n72 to graph
added node n73 to graph
added node n74 to graph
added node n75 to graph
added node n76 to graph
added node n77 to graph
added node n78 to graph
added node n79 to graph
added node n80 to graph
added node n81 to graph
added node n82 to graph
added node n83 to graph
added node n84 to graph
added node n85 to graph
added node n86 to graph
added node n87 to graph
added node n88 to graph
added node n89 to graph
added node n90 to graph
added node n91 to graph
added node n92 to graph
added node n93 to graph
added node n94 to graph
added node n95 to graph
added node n96 to graph
added node n97 to graph
added node n98 to graph
added node n99 to graph
added node n100 to graph
graph contains: n1 n10 n100 n11 n12 n13 n14 n15 n16 n17 n18 n19 n2 n20 n21 n22 n23 n24 n25 n26 n27 n28 n29 n3 n30 n31 n32 n33 n34 n35 n36 n37 n38 n39 n4 n40 n41 n42 n43 n44 n45 n46 n47 n48 n49 n5 n50 n51 n52 n53 n54 n55 n56 n57 n58 n59 n6 n60 n61 n62 n63 n64 n65 n66 n67 n68 n69 n7 n70 n71 n72 n73 n74 n75 n76 n77 n78 n79 n80 n81 n82 n83 n84 n85 n86 n87 n88 n89 n9 n90 n91 n92 n93 n94 n95 n96 n97 n98 n99